src/test/java/io/strimzi/kafka/metrics/KafkaMetricsCollectorTest.java

```diff
@@ -22,21 +21,14 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-
 public class KafkaMetricsCollectorTest {
 
     private final MetricConfig metricConfig = new MetricConfig();
     private final Time time = Time.SYSTEM;
-    private Map<String, String> labels;
-
-    @BeforeEach
-    public void setup() {
-        labels = new HashMap<>();
-        labels.put("key", "value");
-    }
+    private final Map<String, String> labels = Map.of("key", "value");
```

Comment:
Since you're making this change here, do you want to do the same in YammerMetricsCollectorTest? as it has the exact same code.

Reply:
We decided to keep as is because the ordering is clear

Reply:
the reply doesn't really make sense, but I'm not too worried either way.

Reply:
Sorry, wrong place for my last comment.

Reply:
I'd still prefer to use the same logic in both `KafkaMetricsCollectorTest` and `YammerMetricsCollectorTest`. So either we revert this, or we do the same in `YammerMetricsCollectorTest`.

Reply:
https://github.com/strimzi/metrics-reporter/pull/21/files#r1644743277 has not been addressed

Reply:
Sorry, I missed this comment somehow. To keep uniformity, I will revert `KafkaMetricsReporter` to the former logic. 
In trying to implement `private final LinkedHashMap<String, String> tags = new LinkedHashMap<>(Map.of("key", "value", "key2", "value2"));
`, while the LinkedHashMap provided ordering guarantee, I think the Map.of method creates an immutable map, and using it directly to initialize a mutable LinkedHashMap might cause issues in some cases, causing the test to fail sometimes.